docs/src/faq.md

```diff
@@ -32,7 +34,7 @@ julia> last(warsaw.transitions)
 2037-10-25T01:00:00 UTC+1/+0 (CET)
 
 julia> warsaw.cutoff  # DateTime up until the last transition is effective
-Nullable{DateTime}(2038-03-28T01:00:00)
+Nullable{DateTime}(DateTime(2038, 3, 28, 1))
```

Comment:
Doesn't this printing only apply to Julia master?

Reply:
See the filters at the top

Reply:
Both are accepted

Reply:
Ah, okay, thanks.

Reply:
The new format is only in the Julia master right now. Because of the regex I could use either string here and I think this is probably better.